build.fsx

```diff
@@ -107,6 +107,21 @@ Target "CreatePackageEventFlowReadStoresMsSql" (fun _ ->
             "Source/EventFlow.ReadStores.MsSql/EventFlow.ReadStores.MsSql.nuspec"
     )
 
+Target "CreatePackageEventFlowReadStoresElasticsearch" (fun _ ->
+    let binDir = "Source/EventFlow.ReadStores.Elasticsearch/bin/"
+    CopyFile binDir (binDir + buildMode + "/EventFlow.ReadStores.Elasticsearch.dll")
```

Comment:
@rasmus I fixed that - but lets have a talk about it tomorrow - I think we might need to have NEST referenced as a nuget dependency, since the enduser could use other versions for defining their models. But lets have a talk :)